scipy/_lib/_array_api.py

```diff
@@ -6,8 +6,6 @@
 The SciPy use case of the Array API is described on the following page:
 https://data-apis.org/array-api/latest/use_cases.html#use-case-scipy
 """
-from __future__ import annotations
```

Comment:
Be careful with imports within a `if TYPE_CHECKING: `: they are not available a runtime. But all annotations are going to be eagerly evaluated in this module now...

Reply:
I've removed the `TYPE_CHECKING` condition here